src/counter/combined.rs

```diff
@@ -0,0 +1,94 @@
+use chrono::{DateTime, Utc};
+use tokio::sync::mpsc::{UnboundedReceiver, UnboundedSender};
+use ya_gsb_http_proxy::monitor::{RequestsMonitor, ResponseMonitor};
+
+use super::SharedCounters;
+
+#[derive(Clone, Debug)]
+pub(super) struct RequestsMonitoringCounters {
+    counters: SharedCounters,
+    response_time_tx: UnboundedSender<DateTime<Utc>>,
+}
```

Comment:
Why do you need another object for handling counters?
Can't you just implement traits for `Counters` struct?

Reply:
Maybe I could renamed it similar to name of the module.
I wanted to have a counter implementation combining all counters which would also implement `RequestMonitor` trait.
At the same time I did not wanted to have top level struct `Counters` to implement it (and implementation of pub trait on pub struct cannot be private).